src/ints/mouse.cpp

```diff
@@ -31,21 +34,57 @@
 #include "bios.h"
 #include "dos_inc.h"
 
+using namespace bit::literals;
+
+MouseInfoConfig mouse_config;
+MouseInfoVideo mouse_video;
+
+enum class EventType : uint8_t { // compatible with DOS driver mask in driver function 0x0c
+	NotDosEvent    = 0x00,
+	MouseHasMoved  = 0x01,
+	PressedLeft    = 0x02,
+	ReleasedLeft   = 0x04,
+	PressedRight   = 0x08,
+	ReleasedRight  = 0x10,
+	PressedMiddle  = 0x20,
+	ReleasedMiddle = 0x40,
+	WheelHasMoved  = 0x80,
+};
+
+static uint8_t buttons_12  = 0; // state of buttons 1 (left), 2 (right), as visible on host side
+static uint8_t buttons_345 = 0; // state of mouse buttons 3 (middle), 4, and 5 as visible on host side
+
+MouseInfoConfig::MouseInfoConfig() : sensitivity_x(0.3f), sensitivity_y(0.3f) {}
+
+MouseInfoVideo::MouseInfoVideo()
+        : fullscreen(true),
+          res_x(320),
+          res_y(200),
+          clip_x(0),
+          clip_y(0)
+{}
```

Comment:
Move these to default initializers in the struct, and then delete this constructor entirely.